tests/system/Database/Live/EscapeTest.php

```diff
@@ -40,7 +40,7 @@ protected function setUp(): void
      */
     public function testEscapeProtectsNegativeNumbers()
```

Comment:
This should be renamed to say that this does not escape negative numbers now.

Reply:
Fixed.